batavia/types/Tuple.js

```diff
@@ -547,12 +551,20 @@ Tuple.prototype.count = function(value) {
     return count
 }
 
-Tuple.prototype.index = function(value, start, stop) {
-    if (arguments.length < 1) {
-        throw new exceptions.TypeError.$pyclass('index() takes at least 1 argument (' + arguments.length + ' given)')
-    } else if (arguments.length > 3) {
-        throw new exceptions.TypeError.$pyclass('index() takes at most 3 arguments (' + arguments.length + ' given)')
+Tuple.prototype.count.$pyargs = true
```

Comment:
I can see what you're doing here, but I'm not convinced it's the right approach. I think the fix is a little higher up, in the dispatch mechanism itself.